### ISSUE TITLE
Snap packaged app, GTK3 Dialogs not been styled

### ISSUE THREAD

**Issue by eevargas** [electron-userland/electron-builder#2887]
* **Version**:  20.12

* **Target**:  Linux x64 (snap)

* **Electron**: 2.0.0

I have a small electron UI for docker. Using electron-builder to build the snap. When running from the unpacked build. The dialogs look just fine. But when running from the snap package, the dialog does not reflect any GTK3 theme. No drop-shadow, border or style at all. Does anyone know if this is an electron-builder issue or a snap package issue? BTW, I’m, using classic confinement.

### My build configuration:
```
  "build": {
    "appId": "us.evargas.gangway",
    "directories": {
      "buildResources": "icons",
      "app": "build"
    },
    "linux": {
      "target": [
        "snap"
      ],
      "icon": "icon.png",
      "category": "Utilities"
    },
    "snap": {
      "confinement": "classic",
      "grade": "devel"
    }
  }
```

![screenshot](https://nextcloud.evargas.us/index.php/apps/files_sharing/ajax/publicpreview.php?x=1493&y=547&a=true&file=bitmap.png&t=ckmfUY9xmPicg2J&scalingup=0)

**Comment by develar** [comment]
Please try 20.14.7. What version of Ubuntu do you use?

**Comment by eevargas** [comment]
That is on 18.04

**Comment by eevargas** [comment]
Tried 20.14.7... same result.

**Comment by develar** [comment]
Fixed in 20.44.0.
